go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/aws-observability/aws-otel-collector
 go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.44.232
 	github.com/open-telemetry/opentelemetry-collector-contrib/confmap/provider/s3provider v0.75.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter v0.75.0
```

Comment:
All collector (contrib and core) versions need to be updated to the latest at the same time.

Reply:
I have fixed this and made them match